stardog/admin.py

```diff
@@ -1080,8 +1080,14 @@ def __init__(self, name, client):
         self.query_name = name
         self.client = client
         self.path = "/admin/queries/stored/{}".format(name)
-        self.details = {}
-        self.__refresh()
+        """ if details is valid type and not empty set details
```

Comment:
This should just be a regular comment and not a doc string, since this object is not meant to be initialized directly we don't need to expose this to the end users.